data-structures-stacks/max-value.js

```diff
@@ -0,0 +1,12 @@
+export default function maxValue(stack) {
+  let maxNumber = -Infinity;
+  if (stack.peek() === undefined) {
+    return -Infinity;
+  }
+
+  while (stack.peek() >= maxNumber) {
```

Comment:
I don't think this works. I'm trying to figure out why the tests pass!

This should stop getting new values once it gets to a smaller value, so it won't get the actual maximum. You need to make sure you pop and examine every value in the stack.

Update: The tests pass because the values it tested didn't hit this case. I just modified it so this code no longer passes. In max-value.test.js, change the numbers `45, 26, 34, 85, 37, 0` to `45, 26, 34, 85, 25, 40, 37, 0` and you'll see the test no longer passes. This will give you something to test against.